src/lib/Handset/handset.h

```diff
@@ -2,6 +2,10 @@
 
 #include "targets.h"
 
+#if defined(PLATFORM_ESP32)
+#include "driver/rmt.h"
+#endif
+
```

Comment:
Not required

Reply:
yes, initialitally I added some RMT specific methods to the handset, but was able to move the use of RMT to the implementation details of the AutoDetect and PPMHandset.

Removed!